test/testuninitvar.cpp

```diff
@@ -6492,6 +6492,10 @@ class TestUninitVar : public TestFixture {
             "    return n;\n"
             "}\n");
         ASSERT_EQUALS("[test.cpp:4] -> [test.cpp:1]: (error) Using argument i that points at uninitialized variable n\n", errout.str());
+
+        ctu("void increment(int& i) { ++i; }\n" // #6475
+            "int f() { int n; increment(n); return n; }\n");
+        ASSERT_EQUALS("[test.cpp:2] -> [test.cpp:1]: (error) Using argument i that points at uninitialized variable n\n", errout.str());
```

Comment:
This test was already added above at line 6475.

Reply:
Added by myself, no less...